docker/docker-jaxrs-dockerfile/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM java:jre-alpine
```

Comment:
I couldn't find this base image on DockerHub.

Is this an Oracle JRE bundled?

If so, is that permissable by license?